api/src/utils/apply-query.ts

```diff
@@ -540,22 +540,23 @@ export function applyFilter(
 			// See https://github.com/knex/knex/issues/4518 @TODO remove as any once knex is updated
 
 			// These operators don't rely on a value, and can thus be used without one (eg `?filter[field][_null]`)
```

Comment:
```suggestion
```
This comment should be removed as this is no longer the case with the validation done in https://github.com/directus/directus/blob/18760f8f56e41de1aef44e7b875c060b467525a5/api/src/utils/validate-query.ts#L45